src/components/Quill/Quill.tsx

```diff
@@ -50,6 +50,10 @@ export class Quill extends React.Component<IProps, IState> {
     this.onMessage = this.onMessage.bind(this);
     this.loadResources();
   }
+  
+  reloadResources = () => {
```

Comment:
```suggestion
  public reload() {
```

In terms of API, I think this would be a more logical method name, agree? Also, it's intended as a public method, so that should be reflected in the TypeScript interface.

Reply:
Yes, it's tested and working